examples/complete/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# https://github.com/cloudposse/geodesic/
+ARG GEODESIC_VERSION=0.147.11
+ARG GEODESIC_OS=debian
+# https://github.com/cloudposse/atmos
+ARG ATMOS_VERSION=1.3.23
+
+FROM cloudposse/geodesic:${GEODESIC_VERSION}-${GEODESIC_OS}
+
+# Geodesic message of the Day
+ENV MOTD_URL="https://geodesic.sh/motd"
+
+# Some configuration options for Geodesic
+ENV AWS_SAML2AWS_ENABLED=false
+ENV AWS_VAULT_ENABLED=false
+ENV AWS_VAULT_SERVER_ENABLED=false
+ENV GEODESIC_TF_PROMPT_ACTIVE=false
+ENV DIRENV_ENABLED=false
+
+# Enable advanced AWS assume role chaining for tools using AWS SDK
+# https://docs.aws.amazon.com/sdk-for-go/api/aws/session/
+ENV AWS_SDK_LOAD_CONFIG=1
+ENV AWS_DEFAULT_REGION=us-east-2
+
+# Install specific versions of Terraform
+ARG TF_1_0_VERSION=1.1.3
+RUN apt-get update && apt-get install -y -u --allow-downgrades \
+  terraform-1="${TF_1_0_VERSION}-*" && \
+  update-alternatives --set terraform /usr/share/terraform/1/bin/terraform
+
+ARG ATMOS_VERSION
+RUN apt-get update && apt-get install -y --allow-downgrades \
+  atmos="${ATMOS_VERSION}-*" \
+  rainbow-text \
+  kubectl
```

Comment:
Do not include `rainbow-text` or `kubectl` in default/example Dockerfile. Users do not need `rainbow-text` and should install a version of `kubectl` that is compatible with their version of Kubernetes.